packages/database/src/core/Constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const FIREBASE_DATABASE_EMULATOR_HOST_VAR =
+  'FIREBASE_DATABASE_EMULATOR_HOST';
```

Comment:
Having a dedicated file for 1 constant seems overly forward-looking. 😛 

Maybe just add it to core/util/util.ts ?